lib/CConfigHandler.cpp

```diff
@@ -67,6 +67,7 @@ void SettingsStorage::init()
 
 	JsonUtils::maximize(config, "vcmi:settings");
 	JsonUtils::validate(config, "vcmi:settings", "settings");
+	originalConfig = config;

```

Comment:
Should we only save "session" here?

Reply:
Would be sufficient - I figured we might need whole config at some point in the future, but that's perhaps premature.

Reply:
No session parameters should be in settings at all, but some more settings could be added to CLI.

Reply:
I haven't used any of the session parameters myself yet (apart from `savefrequency`), but I figured that they were added to sessions because there was a need to configure them on the fly (e.g. to disable autosaves for faster mass-testing).
Do you mean launcher UI rather than CLI? Or to modify non-session parameters from CLI (without saving)?